cfy_manager/main.py

```diff
@@ -632,6 +631,62 @@ def _create_component_objects(include_components):
         )
 
 
+def _get_components(include_components=None):
+    """Get the component objects based on the config.
+
+    This looks at the config, and returns only the component objects
+    that are supposed to be installed(/configured/started).
+    """
+    c = []
```

Comment:
if you can figure out a better name for this, let me know

Reply:
'required' or 'required_components'?

Reply:
but they're not really required, are they :/
they're even filtered out later

Reply:
current_components?

Reply:
I mean, what are they? Configured ones?